src/components/ItemsList.js

```diff
@@ -104,12 +110,22 @@ const ItemsList = () => {
                   .toLowerCase()
                   .includes(searchTerm.toLowerCase()),
               )
+              .sort(sortShoppingList)
```

Comment:
Interesting - for some reason, I was thinking that sorting the items alphabetically would be really complex?? Glad to see this!